Sources/UI/Camera/PhotoCamera.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  PhotoCamera.swift
+//  PovioKit
+//
+//  Created by Ndriqim Nagavci on 13/10/2022.
+//  Copyright © 2022 Povio Inc. All rights reserved.
+//
+
+import AVFoundation
+import UIKit
+
+public protocol PhotoCameraDelegate: AnyObject {
+  func photoCameraDidTakePhoto(_ image: UIImage)
+  func photoCameraDidTriggerError(_ error: Camera.Error)
+}
+
+public class PhotoCamera: Camera {
+  public weak var delegate: PhotoCameraDelegate?
+  private let photoOutput = AVCapturePhotoOutput()
+  private var deviceInput: AVCaptureDeviceInput?
+  
+  public init(delegate: PhotoCameraDelegate? = nil) {
+    super.init()
+    self.delegate = delegate
+  }
+}
+
+// MARK: - Public Methods
+public extension PhotoCamera {
+  func prepare() {
+    sessionQueue.async {
+      do {
+        try self.configureComponents()
+      } catch {
+        
+      }
+    }
+  }
```

Comment:
What happens on error?

Reply:
I was thinking of adding a completion block
`func prepare(completion: ((Error?) -> Void)? = nil)`

This way we make sure the user knows if the configuration failed for any reason.
The other approach would be to log the error, but I would prefer adding completion. Thoughts?

Reply:
Since `session.startRunning()` is a blocking call and `configureComponents` must be called before that, I think it's safer to have this also in the same thread.  We can make the method async.